docs/support/faq/how-to-resize-in-pouta.md

```diff
@@ -6,12 +6,19 @@ It's possible to proceed by taking a snapshot of your instance and then boot the
 You can find more information about snapshot [here](../../cloud/pouta/snapshots.md) and how to proceed.
 
 ### Using the resize functionality 
+
+Few points regarding the resize functionality:
+
+- [I/O flavors](../../cloud/pouta/vm-flavors-and-billing.md#io-flavors_2) and [GPU flavors](../../cloud/pouta/vm-flavors-and-billing.md#gpu-flavors_2) cannot be resized to a different family flavors. I/O flavors use an ephemeral disk. The data that are not volumes are not copied when the resize functionality is used. This means that your machine won't be able to boot and the message "No bootable device" will be displayed in the console. We recommend to use [snapshots](../../cloud/pouta/snapshots.md#launching-a-snapshot-from-the-web-interface).
+
+- You can safely use the resize process if your I/O flavor vm is booted from a volume.
+
 !!! Warning    
-    [I/O flavors](../../cloud/pouta/vm-flavors-and-billing.md#io-flavors_2) and [GPU flavors](../../cloud/pouta/vm-flavors-and-billing.md#gpu-flavors_2) cannot be resized to a different family flavors. We recommend to use [snapshots](../../cloud/pouta/snapshots.md#launching-an-instance-from-a-volume-snapshot).
+    Keep in mind that when creating a snapshot, only the contents of the root disk will be preserved. If you have data located in the `/mnt` directory for example, back it up in `Allas` using a tool such as [rclone](../../data/Allas/using_allas/rclone.md) or [s3cmd](../../data/Allas/using_allas/s3_client.md)
```

Comment:
- `If you have data located in the /mnt directory...` -> `If you have data in the ephemeral disk, which is typically found in the /mnt directory, ...` but this is more up to you
- `back it up in Allas...` -> I suppose a perhaps easier approach is to back up the data in a cinder volume. Allas is anyway an option too